dpc-go/dpc-api/src/client/ssas_client.go

```diff
@@ -50,6 +51,25 @@ func NewSsasHTTPClient(config SsasHTTPClientConfig) SsasClient {
 	}
 }
 
+// GetSystem function to get a ssas system
+func (sc *SsasHTTPClient) GetSystem(ctx context.Context, systemID string) (GetSystemResponse, error) {
+	log := logger.WithContext(ctx)
+
+	url := fmt.Sprintf("%s/%s/%s", sc.config.URL, PostV2SystemEndpoint, systemID)
+
+	resBytes, err := sc.doGet(ctx, url)
+	if err != nil {
+		log.Error("Get ssas system request failed", zap.Error(err))
```

Comment:
When we're writing text, let's consistently capitalize acronyms. There are several other ocurrences.
```suggestion
		log.Error("Get SSAS system request failed", zap.Error(err))
```